vba_timeseries/models/exp/m_transform_phi.m

```diff
@@ -0,0 +1,10 @@
+function [phi_trans] = m_transform_phi(phi, inG)
+% Parameter transformation function for parameters in phi (observation function)
+
+phi_trans = [ exp(phi(1)); ... %gamma: positive
```

Comment:
The exponential transform to enforce positivity seems fine to me. Your ~N(0, 10) prior on the parameter will mean that the internal values of VBA will often be negative to get to reasonable values like exp(-2). But that's okay. If we start to see weird model fitting performance, we might try the gamma distribution transform, like with phi where we do the normal -> gamma CDF transform.